pytket/binders/circuit/Circuit/main.cpp

```diff
@@ -415,14 +415,7 @@ void def_circuit(py::class_<Circuit, std::shared_ptr<Circuit>> &pyCircuit) {
           ":return: the number of gates in the Circuit")
       .def_property_readonly(
           "wasm_uid",
-          [](const Circuit &circ) {
-            std::optional<std::string> result;
-            try {
-              result = circ.get_wasm_file_uid();
-            } catch (const std::exception &) {
-            }
-            return result;
-          },
+          [](const Circuit &circ) { return circ.get_wasm_file_uid(); },
```

Comment:
```suggestion
          &Circuit.get_wasm_file_uid,
```

Reply:
I was not able to get this to work, I already tried a few things before I saw the suggestion, and I think `[](const Circuit &circ) { return circ.get_wasm_file_uid(); },` is the best way to do it.